package.json

```
@@ -37,25 +37,20 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "asmcrypto.js": "^2.3.2",
-    "bn.js": "^5.0.0",
-    "browserify-aes": "^1.2.0",
-    "bs58": "^4.0.1",
+    "buffer": "^5.5.0",
     "err-code": "^2.0.0",
     "iso-random-stream": "^1.1.0",
     "keypair": "^1.0.1",
-    "libp2p-crypto-secp256k1": "~0.4.0",
-    "multihashing-async": "~0.8.0",
+    "libp2p-crypto-secp256k1": "libp2p/js-libp2p-crypto-secp256k1#fix/add-buffer",
```

libp2p-crypto-secp256k1 has been released
```suggestion
    "libp2p-crypto-secp256k1": "^0.4.2",
```